libmscore/articulation.cpp

```diff
@@ -298,6 +310,13 @@ void Articulation::write(Xml& xml) const
       xml.tag("subtype", subtypeName());
       if (_timeStretch != 1.0)
             xml.tag("timeStretch", _timeStretch);
+      writeProperty(xml, P_ID::PLAY_ARTICULATION);
+      // avoid writing an integer for ornamentStyle, write a string instead.
+      if ( ornamentStyle() == MScore::OrnamentStyle::DEFAULT) {
+          ; // do nothing
+      } else if ( ornamentStyle() == MScore::OrnamentStyle::BAROQUE ) {
+          xml.tag("ornamentStyle", "Baroque");
```

Comment:
Why not using writeProperty here? It should deal with the "do nothing if it's default"